.github/workflows/build.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       matrix:
         target:
           - hoprd
+          - hoprd-debug
```

Comment:
It'd be best if this was not built for each PR, but only when the PR is merged.

Reply:
Really? I actually think its valuable during PR development and after.